game-core/src/main/java/games/strategy/util/Util.java

```diff
@@ -71,4 +74,23 @@ public static String sha512(final String input) {
 
     return p.negate();
   }
+
+  public static String getFromSetting(final ClientSetting<char[]> setting) {
```

Comment:
This method would probably be more appropriate in the `g.s.triplea.settings` package.

It should probably also accept `GameSetting` instead of `ClientSetting`.  That will make it easier to unit test.